<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update</class>
 <widget class="QDialog" name="update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="load_modifier_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_modifier_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>91</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>référence</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>salaire</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>expérience</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>nom de société</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>poste de travail</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>niveau scolaire</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>139</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="reference_modifier_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="salaire_modifier_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="experience_modifier_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nom_modifier_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="poste_modifier_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="niveau_modifier_3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>update</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>update</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
